README.md

```diff
@@ -134,37 +134,37 @@ Windows.
 
 1.  Install [Vagrant](https://www.vagrantup.com/downloads.html) and [docker](https://docs.docker.com/engine/getstarted/step_one/). Both tools support Linux, MacOS and Windows.
 
-2.  Clone this code repository:
+1.  Clone this code repository:
```

Comment:
Why do you change the numbers? The plain markdown file is less readable this way.
Markdown aims to have a readable plain text file as long as a prettified rendered document. ;)

Reply:
You are right, it is a bit less readable. Using "1." for every number makes it easier to change the list. If you want to insert/remove a step in the middle, you don't have to renumber everything below (technically you don't have to do it even when using a higher number, but then it gets confusing). 

Using "1." makes it clear that this is a ordered list, if you want to edit it that is enough (you just care about the order itself, not which number a step has).

For just viewing, I recommend the rendered version if you care about its beauty. Reading the plain version of this is still good enough as a step by step checklist in my opinion (here we have the commands that take up 3 lines and make good "anchors").